infrastructure/builder/src/release/tasks.js

```diff
@@ -105,7 +105,15 @@ module.exports = ({tasks, cmdOptions}) => {
         contents.replace(/VERSION = .*/, `VERSION = '${requirements['release-version']}'`));
       changed.push(pyclient);
 
-      // go client is versioned only by git tag
+      // the go client requires the major version number in its import path, so
+      // just about every file needs edited.  This matches the full package
```

Comment:
```suggestion
      // just about every file needs to be edited.  This matches the full package
```

Reply:
I was trying to reflect the Pittsburgh manner of speaking :)